scripts/docker/run_headless.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
Could we make this part of run.sh? Like add an option --headless

Reply:
There are other differences between the two scripts.

I wrote them separately this way because I was trying to avoid breaking the existing `run.sh` in case there are scripts that count on its exact current behavior.

It would not be hard to create a combo script that can do the jobs of both `run.sh` and `run_headless.sh`, but I would ideally want to have a free hand to choose the argument semantics that make the most sense across the wider range of use cases, without requiring backward compatibility with the current `run.sh` arguments. Do you think writing a combo script like that would be worthwhile?

Reply:
In my opinion it's worth it to go through the trouble of merging it maintaining backwards compatibility. We kept in the past multiple run scripts in the isaac docker folder and it became hard to maintain when changes were introduced (we since then merged them all under one run.sh). I'll leave it up though :)

Reply:
I agree, I think it would be worthwhile to merge the two scripts and save us headache in the future.

Reply:
Ok, merged `run_headless.sh` features into `run.sh` in e4cf0fd96f757edc8282cff45a8f719a872e4311

The intent is explained at [1]. It changes the argument semantics of `run.sh` slightly for the original use case. (You need to specify `--sim` to run the simulator, and `--sim-args` instead of `--args`.) Otherwise, it just adds useful features.

[1] https://github.com/trey0/astrobee/blob/docker_testing/scripts/docker/readme.md#run-commands-in-the-container